scripts/46-audio-receiver.sh

```diff
@@ -0,0 +1,14 @@
+# Raspotify installation script
```

Comment:
Probably should have a `#!` to match other files (though I don't think Debian ever uses `sh` instead of `bash` by default)